lib/broccoli/angular2-app.js

```diff
@@ -87,13 +99,80 @@ Angular2App.prototype.toTree = function () {
     destDir: 'vendor'
   });
 
-  var merged = mergeTrees([
+  var sassSrcTree = new Funnel(sourceDir, {
+    include: ['**/*.sass', '**/*.scss'],
+    allowEmpty: true
+  });
+
+  var lessSrcTree = new Funnel(sourceDir, {
+    include: ['**/*.less'],
+    allowEmpty: true
+  });
+
+  var stylusSrcTree = new Funnel(sourceDir, {
+    include: ['**/*.styl'],
+    allowEmpty: true
+  });
+
+  var trees = [
     assetTree,
     tsTree,
     jsTree,
     this.index(),
     vendorNpmTree
-  ], { overwrite: true });
+  ];
+
+  try {
+    if (process.platform === 'win32') {
```

Comment:
That's way too much code. Can't you just ```
// 
trees = trees.concat(
  require('./angular-broccoli-sass').makeBroccoliTree(sourceDir),
  require('./angular-broccoli-less').makeBroccoliTree(sourceDir),
  require('./angular-broccoli-compass').makeBroccoliTree(sourceDir)
]).filter(x => !!x);
```

etc etc.  The `broccoli-sass` script should make the Funnel and check if sass is installed. If not, `return null`.

What do you think?